### DISCUSSION TITLE
Problem with bcftools

### DISCUSSION THREAD

**Discussion by JoeSecGen** [biostars/biostar-handbook-issues#293] [General]
Hello,
Firstly, I want to thank to the creators of biostar-handbook, It is an useful book for principiants in bioinformatics, like me..
Well, I had followed all step for the installation of homebrew, miniconda and channels and command line to get the tools bioinformatics, but I have one problem with bcftools, it is the only tool that I can't execute in my environment, when I write bcftools in my terminal, I see this:
(Secuenciacion_biomol) apple@Joel Documents % bcftools
dyld[5059]: Library not loaded: '@rpath/libquadmath.0.dylib'
  Referenced from: '/Users/apple/miniconda3/envs/Secuenciacion_biomol/lib/libgfortran.5.dylib'
  Reason: tried: '/Users/apple/miniconda3/envs/Secuenciacion_biomol/lib/libquadmath.0.dylib' (no such file), '/Users/apple/miniconda3/envs/Secuenciacion_biomol/lib/libquadmath.0.dylib' (no such file), '/Users/apple/miniconda3/envs/Secuenciacion_biomol/lib/libquadmath.0.dylib' (no such file), '/Users/apple/miniconda3/envs/Secuenciacion_biomol/bin/../lib/libquadmath.0.dylib' (no such file), '/Users/apple/miniconda3/envs/Secuenciacion_biomol/bin/../lib/libquadmath.0.dylib' (no such file), '/usr/local/lib/libquadmath.0.dylib' (no such file), '/usr/lib/libquadmath.0.dylib' (no such file)
zsh: abort      bcftools

I understand that the problem is the libquadmath, I had reinstalled all the step but I can't find out a solution for that, I don't know if I can install only the lib or maybe is the version of my homebrew, because I have Mac OS Monterrey 12.7 and Homebrew 4.1.16

Any suggestions I will be grateful

Joel

**Comment by ialbert** [comment]
I found that these problems are usually caused by the wrong order of the channels.

conda forge needs to be first then bioconda

uninstall bcftools then the command would be something like

   conda config --prepend channels conda-forge
   conda config --append channels bioconda
   conda config --set channel_priority strict

then reinstall bcftools

note that the new edition of the book uses micromamba and all installation works through that. 

micromamba is a better version of conda

see the new installation instructions in the web version of the book, the PDF distributed as a download is still the old version
